### ISSUE TITLE
1.20.6 broke everything

### ISSUE THREAD

**Issue by 0PandaDEV** [TriumphTeam/triumph-gui#117]
All the guis no longer work it throws errors on mass.

```log
[06:38:24 INFO]: PinguLovesU issued server command: /menu
[06:38:25 WARN]: java.lang.ClassNotFoundException: org.bukkit.craftbukkit.craftbukkit.inventory.CraftMetaItem
[06:38:25 WARN]:        at org.bukkit.plugin.java.PluginClassLoader.loadClass0(PluginClassLoader.java:197)
[06:38:25 WARN]:        at org.bukkit.plugin.java.PluginClassLoader.loadClass(PluginClassLoader.java:164)
[06:38:25 WARN]:        at java.base/java.lang.ClassLoader.loadClass(ClassLoader.java:526)
[06:38:25 WARN]:        at java.base/java.lang.Class.forName0(Native Method)
[06:38:25 WARN]:        at java.base/java.lang.Class.forName(Class.java:534)
[06:38:25 WARN]:        at java.base/java.lang.Class.forName(Class.java:513)
[06:38:25 WARN]:        at io.papermc.reflectionrewriter.runtime.AbstractDefaultRulesReflectionProxy.forName(AbstractDefaultRulesReflectionProxy.java:68)
[06:38:25 WARN]:        at io.papermc.paper.pluginremap.reflect.PaperReflectionHolder.forName(Unknown Source)
[06:38:25 WARN]:        at Nextron-1.5.5.jar//dev.triumphteam.gui.components.util.VersionHelper.craftClass(VersionHelper.java:143)
[06:38:25 WARN]:        at Nextron-1.5.5.jar//dev.triumphteam.gui.builder.item.BaseItemBuilder.<clinit>(BaseItemBuilder.java:77)
[06:38:25 WARN]:        at Nextron-1.5.5.jar//net.pandadev.nextron.guis.GUIs.mainGui(GUIs.java:29)
[06:38:25 WARN]:        at Nextron-1.5.5.jar//net.pandadev.nextron.commands.MenuCommand.execute(MenuCommand.java:26)
[06:38:25 WARN]:        at Nextron-1.5.5.jar//net.pandadev.nextron.commands.CommandBase.onCommand(CommandBase.java:50)
[06:38:25 WARN]:        at org.bukkit.command.PluginCommand.execute(PluginCommand.java:45)
[06:38:25 WARN]:        at io.papermc.paper.command.brigadier.bukkit.BukkitCommandNode$BukkitBrigCommand.run(BukkitCommandNode.java:86)
[06:38:25 WARN]:        at com.mojang.brigadier.context.ContextChain.runExecutable(ContextChain.java:73)
[06:38:25 WARN]:        at net.minecraft.commands.execution.tasks.ExecuteCommand.execute(ExecuteCommand.java:31)
[06:38:25 WARN]:        at net.minecraft.commands.execution.tasks.ExecuteCommand.execute(ExecuteCommand.java:19)
[06:38:25 WARN]:        at net.minecraft.commands.execution.UnboundEntryAction.lambda$bind$0(UnboundEntryAction.java:8)
[06:38:25 WARN]:        at net.minecraft.commands.execution.CommandQueueEntry.execute(CommandQueueEntry.java:5)
[06:38:25 WARN]:        at net.minecraft.commands.execution.ExecutionContext.runCommandQueue(ExecutionContext.java:103)
[06:38:25 WARN]:        at net.minecraft.commands.Commands.executeCommandInContext(Commands.java:442)
[06:38:25 WARN]:        at net.minecraft.commands.Commands.performCommand(Commands.java:349)
[06:38:25 WARN]:        at net.minecraft.commands.Commands.performCommand(Commands.java:336)
[06:38:25 WARN]:        at net.minecraft.commands.Commands.performCommand(Commands.java:331)
[06:38:25 WARN]:        at net.minecraft.server.network.ServerGamePacketListenerImpl.performUnsignedChatCommand(ServerGamePacketListenerImpl.java:2202)
[06:38:25 WARN]:        at net.minecraft.server.network.ServerGamePacketListenerImpl.lambda$handleChatCommand$14(ServerGamePacketListenerImpl.java:2176)
[06:38:25 WARN]:        at net.minecraft.server.TickTask.run(TickTask.java:18)
[06:38:25 WARN]:        at net.minecraft.util.thread.BlockableEventLoop.doRunTask(BlockableEventLoop.java:151)
[06:38:25 WARN]:        at net.minecraft.util.thread.ReentrantBlockableEventLoop.doRunTask(ReentrantBlockableEventLoop.java:24)
[06:38:25 WARN]:        at net.minecraft.server.MinecraftServer.doRunTask(MinecraftServer.java:1511)
[06:38:25 WARN]:        at net.minecraft.server.MinecraftServer.doRunTask(MinecraftServer.java:195)
[06:38:25 WARN]:        at net.minecraft.util.thread.BlockableEventLoop.pollTask(BlockableEventLoop.java:125)
[06:38:25 WARN]:        at net.minecraft.server.MinecraftServer.pollTaskInternal(MinecraftServer.java:1488)
[06:38:25 WARN]:        at net.minecraft.server.MinecraftServer.pollTask(MinecraftServer.java:1411)
[06:38:25 WARN]:        at net.minecraft.util.thread.BlockableEventLoop.managedBlock(BlockableEventLoop.java:135)
[06:38:25 WARN]:        at net.minecraft.server.MinecraftServer.waitUntilNextTick(MinecraftServer.java:1377)
[06:38:25 WARN]:        at net.minecraft.server.MinecraftServer.runServer(MinecraftServer.java:1238)
[06:38:25 WARN]:        at net.minecraft.server.MinecraftServer.lambda$spin$0(MinecraftServer.java:323)
[06:38:25 WARN]:        at java.base/java.lang.Thread.run(Thread.java:1583)
[06:38:25 ERROR]: Command exception: /menu
org.bukkit.command.CommandException: Unhandled exception executing command 'menu' in plugin Nextron v1.5.5
        at org.bukkit.command.PluginCommand.execute(PluginCommand.java:47) ~[paper-api-1.20.6-R0.1-SNAPSHOT.jar:?]
        at io.papermc.paper.command.brigadier.bukkit.BukkitCommandNode$BukkitBrigCommand.run(BukkitCommandNode.java:86) ~[paper-1.20.6.jar:1.20.6-112-347bbe3]
        at com.mojang.brigadier.context.ContextChain.runExecutable(ContextChain.java:73) ~[brigadier-1.2.9.jar:?]
        at net.minecraft.commands.execution.tasks.ExecuteCommand.execute(ExecuteCommand.java:31) ~[paper-1.20.6.jar:1.20.6-112-347bbe3]
        at net.minecraft.commands.execution.tasks.ExecuteCommand.execute(ExecuteCommand.java:19) ~[paper-1.20.6.jar:1.20.6-112-347bbe3]
        at net.minecraft.commands.execution.UnboundEntryAction.lambda$bind$0(UnboundEntryAction.java:8) ~[paper-1.20.6.jar:1.20.6-112-347bbe3]
        at net.minecraft.commands.execution.CommandQueueEntry.execute(CommandQueueEntry.java:5) ~[paper-1.20.6.jar:1.20.6-112-347bbe3]
        at net.minecraft.commands.execution.ExecutionContext.runCommandQueue(ExecutionContext.java:103) ~[paper-1.20.6.jar:1.20.6-112-347bbe3]
        at net.minecraft.commands.Commands.executeCommandInContext(Commands.java:442) ~[paper-1.20.6.jar:1.20.6-112-347bbe3]
        at net.minecraft.commands.Commands.performCommand(Commands.java:349) ~[paper-1.20.6.jar:1.20.6-112-347bbe3]
        at net.minecraft.commands.Commands.performCommand(Commands.java:336) ~[paper-1.20.6.jar:1.20.6-112-347bbe3]
        at net.minecraft.commands.Commands.performCommand(Commands.java:331) ~[paper-1.20.6.jar:1.20.6-112-347bbe3]
        at net.minecraft.server.network.ServerGamePacketListenerImpl.performUnsignedChatCommand(ServerGamePacketListenerImpl.java:2202) ~[paper-1.20.6.jar:1.20.6-112-347bbe3]
        at net.minecraft.server.network.ServerGamePacketListenerImpl.lambda$handleChatCommand$14(ServerGamePacketListenerImpl.java:2176) ~[paper-1.20.6.jar:1.20.6-112-347bbe3]
        at net.minecraft.server.TickTask.run(TickTask.java:18) ~[paper-1.20.6.jar:1.20.6-112-347bbe3]
        at net.minecraft.util.thread.BlockableEventLoop.doRunTask(BlockableEventLoop.java:151) ~[paper-1.20.6.jar:1.20.6-112-347bbe3]
        at net.minecraft.util.thread.ReentrantBlockableEventLoop.doRunTask(ReentrantBlockableEventLoop.java:24) ~[paper-1.20.6.jar:1.20.6-112-347bbe3]
        at net.minecraft.server.MinecraftServer.doRunTask(MinecraftServer.java:1511) ~[paper-1.20.6.jar:1.20.6-112-347bbe3]
        at net.minecraft.server.MinecraftServer.doRunTask(MinecraftServer.java:195) ~[paper-1.20.6.jar:1.20.6-112-347bbe3]
        at net.minecraft.util.thread.BlockableEventLoop.pollTask(BlockableEventLoop.java:125) ~[paper-1.20.6.jar:1.20.6-112-347bbe3]
        at net.minecraft.server.MinecraftServer.pollTaskInternal(MinecraftServer.java:1488) ~[paper-1.20.6.jar:1.20.6-112-347bbe3]
        at net.minecraft.server.MinecraftServer.pollTask(MinecraftServer.java:1411) ~[paper-1.20.6.jar:1.20.6-112-347bbe3]
        at net.minecraft.util.thread.BlockableEventLoop.managedBlock(BlockableEventLoop.java:135) ~[paper-1.20.6.jar:1.20.6-112-347bbe3]
        at net.minecraft.server.MinecraftServer.waitUntilNextTick(MinecraftServer.java:1377) ~[paper-1.20.6.jar:1.20.6-112-347bbe3]
        at net.minecraft.server.MinecraftServer.runServer(MinecraftServer.java:1238) ~[paper-1.20.6.jar:1.20.6-112-347bbe3]
        at net.minecraft.server.MinecraftServer.lambda$spin$0(MinecraftServer.java:323) ~[paper-1.20.6.jar:1.20.6-112-347bbe3]
        at java.base/java.lang.Thread.run(Thread.java:1583) ~[?:?]
Caused by: java.lang.ExceptionInInitializerError
        at Nextron-1.5.5.jar/net.pandadev.nextron.guis.GUIs.mainGui(GUIs.java:29) ~[Nextron-1.5.5.jar:?]
        at Nextron-1.5.5.jar/net.pandadev.nextron.commands.MenuCommand.execute(MenuCommand.java:26) ~[Nextron-1.5.5.jar:?]
        at Nextron-1.5.5.jar/net.pandadev.nextron.commands.CommandBase.onCommand(CommandBase.java:50) ~[Nextron-1.5.5.jar:?]
        at org.bukkit.command.PluginCommand.execute(PluginCommand.java:45) ~[paper-api-1.20.6-R0.1-SNAPSHOT.jar:?]
        ... 26 more
Caused by: dev.triumphteam.gui.components.exception.GuiException: Could not retrieve displayName nor lore field for ItemBuilder.
        at Nextron-1.5.5.jar/dev.triumphteam.gui.builder.item.BaseItemBuilder.<clinit>(BaseItemBuilder.java:86) ~[Nextron-1.5.5.jar:?]
        at Nextron-1.5.5.jar/net.pandadev.nextron.guis.GUIs.mainGui(GUIs.java:29) ~[Nextron-1.5.5.jar:?]
        at Nextron-1.5.5.jar/net.pandadev.nextron.commands.MenuCommand.execute(MenuCommand.java:26) ~[Nextron-1.5.5.jar:?]
        at Nextron-1.5.5.jar/net.pandadev.nextron.commands.CommandBase.onCommand(CommandBase.java:50) ~[Nextron-1.5.5.jar:?]
        at org.bukkit.command.PluginCommand.execute(PluginCommand.java:45) ~[paper-api-1.20.6-R0.1-SNAPSHOT.jar:?]
        ... 26 more

```

**Comment by maurxce** [comment]
See https://github.com/TriumphTeam/triumph-gui/issues/114

**Comment by LichtHund** [comment]
Fixed on 3.1.8.

**Comment by Rapha149** [comment]
Is it normal that the new version is not published on Maven Central yet? If so, do you happen to have an approximate ETA on that?

**Comment by LichtHund** [comment]
> Is it normal that the new version is not published on Maven Central yet? If so, do you happen to have an approximate ETA on that?

Sorry for the delay, it should be available now.

**Comment by Rapha149** [comment]
Thank you!

**Comment by LichtHund** [comment]
Turns out I forgot to update adventure on 3.1.8 so I just released 3.1.9 which should be all fixed.
